setup.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import setuptools
+
+with open("README.md",'r') as f:
+    long_description = f.read()
+
+setuptools.setup(
+        name="martini-garnish",
+        version="0.2-alpha",
+        author=["Matthijs Tadema", "Lorenzo Gaifas"],
+        author_email="M.J.Tadema@protonmail.com",
```

Comment:
Please add your email!

Reply:
done already, hopefully in a list it works ok!